### DISCUSSION TITLE
Problem with adding my first customer account

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#9775] [Q&A / Support]
I'm trying to add my first customer within the admin interface of Pimcore 10.0.7 but get this message whenever I press the new customer button. Am I missing something or is there an issue within Pimcore? The url it's using that returns a 404-message is this: http://localhost:9000/admin/object/get?_dc=1626351155305&id=0
![image](https://user-images.githubusercontent.com/5517212/125786283-bb55b3ae-1ba1-4fdd-8d84-e34a742d466d.png)

The error message is in Swedish and basically says: A temporary error occurred - could not complete action. Try reloading the admin interface and try again

**Comment by dvesh3** [comment]
@TS1997 Are you referring to Customer Management Framework Bundle? Could you please share exacts steps to reproduce the problem? thanks!

**Reply by TS1997** [reply]
@dvesh3 Yes, it's the CMF bundle. I installed it and imported the example customer.json as a dataoject class. Then I went into the customers tab and pressed add new customer and got this issue mentioned above. I can also mention that I've made sure everything is at the latest version through composer. Another thing to mention is that the site has been updated from Pimcore 6.4 -> 6.9 -> 10.0.7 but I installed the CMF bundle at 10.0.7

**Reply by dvesh3** [reply]
please make sure you are using CMF bundle version >= 3.1.0 on Pimcore X. It would be helpful, if you can enable debug mode and check logs for errors in /var/log/dev.log.

**Reply by TS1997** [reply]
@dvesh3 I'm running CMF 3.2.0 and Pimcore 10.0.7. I've attached a dev.log file in this comment.
[dev.zip](https://github.com/pimcore/pimcore/files/6828731/dev.zip)

**Reply by TS1997** [reply]
@dvesh3 Do you have any updates in regards to this issue? I've now updated to Pimcore 10.1 and the issue still exists
